packages/spark-core/base/requiredTextInput.js

```diff
@@ -0,0 +1,34 @@
+import getElements from '../utilities/getElements';
+import setValidTextInput from '../utilities/validation/setValidTextInput';
+import setInvalidTextInput from '../utilities/validation/setInvalidTextInput';
+
```

Comment:
This all looks good. We should in the future add some comment blocks describing what the code is for.